package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
```

Comment:
bump `@types/node` (dev dep) to `20.4.4` to solve CI error?

Reply:
On my system, updating `@types/node` produces this error when running `yarn test`:
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/leeyi45/source/source_academy/js-slang/node_modules/wrap-ansi/index.js from /Users/leeyi45/source/source_academy/js-slang/node_modules/cliui/build/index.cjs not supported.
Instead change the require of index.js in /Users/leeyi45/source/source_academy/js-slang/node_modules/cliui/build/index.cjs to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/leeyi45/source/source_academy/js-slang/node_modules/cliui/build/index.cjs:293:14)
    at Object.<anonymous> (/Users/leeyi45/source/source_academy/js-slang/node_modules/yargs/build/index.cjs:1:60678)
    at Object.<anonymous> (/Users/leeyi45/source/source_academy/js-slang/node_modules/yargs/index.cjs:5:30)
    at _yargs (/Users/leeyi45/source/source_academy/js-slang/node_modules/jest-cli/build/run.js:30:39)
    at buildArgv (/Users/leeyi45/source/source_academy/js-slang/node_modules/jest-cli/build/run.js:143:26)
    at Object.run (/Users/leeyi45/source/source_academy/js-slang/node_modules/jest-cli/build/run.js:118:24)
    at Object.<anonymous> (/Users/leeyi45/source/source_academy/js-slang/node_modules/jest-cli/bin/jest.js:16:17)
    at Object.<anonymous> (/Users/leeyi45/source/source_academy/js-slang/node_modules/jest/bin/jest.js:12:3)
```